python/paddle/fluid/contrib/slim/quantization/mkldnn_post_training_strategy.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import logging
+import six
+import numpy as np
+from .... import core
+from ..core.strategy import Strategy
+
+__all__ = ['MKLDNNPostTrainingQuantStrategy']
+
+logging.basicConfig(format='%(asctime)s-%(levelname)s: %(message)s')
+_logger = logging.getLogger(__name__)
+_logger.setLevel(logging.INFO)
+
+
+class MKLDNNPostTrainingQuantStrategy(Strategy):
+    """
+    The strategy for Post Training quantization strategy.
+    """
+
+    def __init__(self, int8_model_save_path=None, fp32_model_path=None):
+        """
+        Args:
+            int8_model_save_path(str): The path to save int8 model with fp32 weights.
```

Comment:
I don't understand what is the `int8 model with fp32 weights`?Please give more detail explanation.

Reply:
The same to me. I comment in https://github.com/PaddlePaddle/Paddle/pull/17634#discussion_r287941929, but it seems still confusing here.

Reply:
It means `MKLDNNPostTrainingQuantStrategy` will save **int8 programdesc + fp32 weights** to the path `int8_model_save_path`

Reply:
> It means `MKLDNNPostTrainingQuantStrategy` will save **int8 programdesc + fp32 weights** to the path `int8_model_save_path`

@zhaify we should make the instructions/explanation in code more clearly.